software/profiling/Vampir.rst

```diff
@@ -1,11 +1,320 @@
 .. _vampir:
 
+.. image:: /images/vampir_logo.png
+   :align: left
+   :width: 230px
+   :height: 130px
+
+|
+|
+|
+|
+|
+|
+
 ************************************
-Vampir (Coming Soon!!)
+Vampir
 ************************************
 
-Vampir is a software performance visualizer focused on highly parallel applications. It presents a unified view on an application run including the use of programming paradigms like MPI, OpenMP, PThreads, CUDA, OpenCL and OpenACC. It also incorporates file I/O, hardware performance counters and other performance data sources. Various interactive displays offer detailed insight into the performance behavior of the analyzed application. Vampir's scalable analysis server and visualization engine enable interactive navigation of large amounts of performance data. Score-P and TAU generate OTF2 trace files for Vampir to visualize.
+Overview
+=========
+
+Vampir is a software performance visualizer focused on highly parallel applications. It presents a unified view on an application run
+including the use of programming paradigms like MPI, OpenMP, PThreads, CUDA, OpenCL and OpenACC.
+It also incorporates file I/O, hardware performance counters and other performance data sources.
+Various interactive displays offer detailed insight into the performance behavior of the analyzed application.
+Vampir's scalable analysis server and visualization engine enable interactive navigation of large amounts of performance data.
+Score-P and TAU generate OTF2 trace files for Vampir to visualize.
+
+
+Usage
+=======
+
+Vampir can be run a few different ways depending on a couple of factors. If you have a large trace file it
+would benefit from utilizing VampirServer to process the trace file with the systems compute power
+while reverse connected to the Vampir GUI. If the trace file is small enough, it would do just fine viewing on a
+login node or your local machine.
+
+.. _vamps:
+
+``Vampirserver`` is the backend software component that can run across multiple compute nodes taking advantage
+of the machines memory, this in turn provides a peak performance for viewing large trace files.
```

Comment:
Again, the possessive form should be used for "machines" (i.e., machine's).
Also, "peak performance" has a specific meaning in our docs, so I would change this to something like "increased performance" or whatever.